profiles/common/modules/features/ec_profiles/ec_profiles.install

```diff
@@ -15,7 +15,7 @@ include_once 'ec_profiles.install.inc';
 function ec_profiles_enable() {
   drupal_set_message(t('EC Profiles feature is now active on your site.'));
```

Comment:
There are 2 activation messages (only 1 is necessary):
``L16: drupal_set_message(t('EC Profiles feature is now active on your site.'));``
``L22: drupal_set_message(t('ec_profiles %v feature is now active on your site.', array('%v' => _ec_profiles_get_version())));``